extractor/src/main/scala/org/jetbrains/sbt/StructureKeys.scala

```diff
@@ -14,6 +14,8 @@ object StructureKeys {
   val testConfigurations: SettingKey[Seq[Configuration]] = SettingKey("ssTestConfigurations", rank = Invisible)
   val acceptedProjects: TaskKey[Seq[ProjectRef]] = TaskKey("ssAcceptedProjects", rank = Invisible)
 
+  val allSourceConfigurations: TaskKey[Seq[Configuration]] = TaskKey("ssAllSourceConfigurations", rank = Invisible)
+  val allTestConfigurations: TaskKey[Seq[Configuration]] = TaskKey("ssAllTestConfigurations", rank = Invisible)
```

Comment:
(minor) Naming:
So right now the names everywhere are like "source" & "test".
I think it would be more consistent to make it like "sources" & "testSource"